src/enhanced-text-input/index.ts

```diff
@@ -21,7 +21,7 @@ export interface EnhancedTextInputProperties extends TextInputProperties {
 		'addonAfter',
 		'addonBefore',
 		'aria',
-		'focus',
+		'shouldFocus',
```

Comment:
`focus` does not blend well when used with custom elements so renamed to `shouldFocus`